CONTRIBUTING.asciidoc

```diff
@@ -526,6 +526,21 @@ generate code and subsequently overwrite part or all of it. Running with all
 will slow Valgrind down noticeably.
 ____
 
+Setting up a Windows Development Environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Install https://www.python.org/ftp/python/3.4.4/python-3.4.4.msi[Python 3.4] (32bit!)
+* Install https://www.qutebrowser.org/pyqt/PyQt5-5.5.1-gpl-Py3.4-Qt5.5.1-x32.exe[PyQt 5.5]
```

Comment:
I'd prefer a link to PyQt's sourceforge package, I mainly have those mirrored to make the CI builds more reliable, but I think we should point people to the upstream download.